test/python/test_with_schedules.py

```diff
@@ -240,6 +240,33 @@ def eigenvector_centrality_verified_test(self, input_file_name, use_separate_alg
         print (lines)
         self.assertEqual(float(lines[0].strip()), 3.2)
 
+    def closeness_centrality_unweighted_test(self, input_file_name, use_separate_algo_file=False):
+	if use_separate_algo_file:
+            self.basic_compile_test_with_separate_algo_schedule_files("closeness_centrality_unweighted.gt", input_file_name)
```

Comment:
Take a look at the functions before you use them. You should not be using a separate_algo_file in this case.